dev-python/pyside/pyside-9999.ebuild

```diff
@@ -135,8 +142,27 @@ src_configure() {
 
 	configuration() {
 		local mycmakeargs=(
+			-DPYTHON_EXECUTABLE="${PYTHON}"
 			"${mycmakeargs[@]}"
 		)
+
+		if python_is_python3; then
+			python_export PYTHON_CONFIG
+
+			# Extension tag unique to the current Python 3.x version (e.g.,
+			# ".cpython-34m" for CPython 3.4).
+			local EXTENSION_TAG="$("${PYTHON_CONFIG}" --extension-suffix)"
+			EXTENSION_TAG="${EXTENSION_TAG%.so}"
+
+			mycmakeargs+=(
+				# Find the previously installed "Shiboken2Config.*.cmake" file
+				# specific to this Python 3.x version.
+				-DPYTHON_EXTENSION_SUFFIX="${EXTENSION_TAG}"
```

Comment:
I don't know why you need all this. `PYTHON_EXTENSION_SUFFIX` seems to be calculated correctly on my system (at least when the aforementioned upstream change is applied).